src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -1795,9 +1796,9 @@ def toIODAobs(self, OutDir, clobber=True):
                 LocVars.append(ncv)
 
         nlocs = self.nobs
-        vname = "ozoneTotal"
+        vname = "ozoneProfile"
```

Comment:
One question: Is this variable defined for ozone measurement with units in mol/mol?

Reply:
yes. MLS and OMPS-LP are profile measurements that are mixing ratio, and not DU.

Reply:
Actually, you're probably right, it should be ppmv, but last I checked the variable mole_fraction_ozone_in_air was in ppmv. Depending on your point of view, this could also be considered wrong.

Reply:
Apologies, I'm not following. Is there something to change?

Reply:
In IODA convention table, there are several ozone-related variables with different units:
- ozoneTotal in DU
- ozoneLayer in DU
- ozoneColumn in mol/m2
- ozoneProfile in mol/mol
- ozoneSurface in kg/kg

So, depending on the ozone obs used, the ozone variables should be named based on their units.

Reply:
I think in the future we could get rid of ozoneSurface as ozoneProfile would be the same, i.e. a vertical interpolation of a point on the geoval profile. To clear the confusion given the comments above, the unit is not that important as DU to **ppmV** (**volume** mixing ratio in mol/mol) to **ppm** (**mass** mixing ratio in kg/kg) is a matter of a single coefficient multiplication. FYI, Some surface measurements, i.e. Europe for ozone are reported in ug/m3. Conversion is then not straightforward and requires P and T within the measurements. This is not that relevant here. Ideally, JEDI should use mixing ratios and not densities. Please try to add people from the JCSDA atmospheric composition team for trace gas and aerosol related PRs. Thanks!